src/test/java/com/qa/koel/Homework18_Play_A_Song.java

```diff
@@ -19,9 +16,10 @@ public void playSong() {
         allSongsPage = new AllSongsPage(getDriver());
 
         loginPage.loginKoel("son.nam.kim@testpro.io","koel08/23");
-        allSongsPage.clickSidePanelListAllSongs();
-        allSongsPage.contextClickFirstSongInAllSongsPlaylist();
-        allSongsPage.clickOptionPlayInDropdownMenu();
+        allSongsPage.clickSidePanelListAllSongs()
+                    .clickFirstSongInAllSongsPlaylist()
+                    .contextClickFirstSongInAllSongsPlaylist()
+                    .clickOptionPlayInDropdownMenu();
 
         Assert.assertTrue(allSongsPage.soundBarsImage().isDisplayed());
     }
```

Comment:
Good practice: class name LoginTests, HomeTests....

Reply:
So instead of naming them AllSongsPage, LoginPage, etc, name them AllSongsTests, LoginTests - am I rightly understood  it?
I think we were instructed to do that way to differ them from the tests file.

Reply:
> So instead of naming them AllSongsPage, LoginPage, etc, name them AllSongsTests, LoginTests - am I rightly understood it? I think we were instructed to do that way to differ them from the tests file.

No, Homework15 - LoginTest and etc...